editor/src/components/editor/editor-component.tsx

```diff
@@ -345,26 +317,27 @@ export const EditorComponentInner = React.memo((props: EditorProps) => {
               alignItems: 'stretch',
             }}
           >
-            <SimpleFlexColumn
-              style={{
-                height: '100%',
-                width: MenuBarWidth,
-                backgroundColor: colorTheme.leftMenuBackground.value,
-              }}
-            >
-              <Menubar />
-            </SimpleFlexColumn>
             <div
               className='LeftPaneShell'
               style={{
                 height: '100%',
+                display: 'flex',
                 flexShrink: 0,
                 transition: IS_TEST_ENVIRONMENT ? 'none' : 'all .1s ease-in-out',
-                width: leftMenuExpanded ? LeftPaneDefaultWidth : 0,
+                width: leftMenuExpanded ? LeftPaneDefaultWidth + MenuBarWidth : 0,
                 overflowX: 'scroll',
                 backgroundColor: colorTheme.leftPaneBackground.value,
               }}
             >
+              <SimpleFlexColumn
+                style={{
+                  height: '100%',
+                  width: MenuBarWidth,
+                  backgroundColor: colorTheme.leftMenuBackground.value,
+                }}
+              >
+                <Menubar />
+              </SimpleFlexColumn>
```

Comment:
Commenting here because I cannot comment directly on the related lines (`473` and onwards) - the `LockerOverlay` component should also be updated to apply the left skew in the style props only when the sidebar is open or it will be incorrectly positioned:

![Screen Shot 2022-11-02 at 1 37 18 PM](https://user-images.githubusercontent.com/1081051/199491389-524320fd-278c-4156-80b1-d5ff7c36c042.png)